client/src/components/geneExpression/geneSet.js

```diff
@@ -0,0 +1,187 @@
+    const { isOpen, toggleSummaryHisto } = this.state;
+    return (
+      <div style={{ marginBottom: 3 }}>
+        <div
+          style={{
+            display: "flex",
+            justifyContent: "space-between",
+            alignItems: "baseline",
+          }}
+        >
+          <span
+            role="menuitem"
+            tabIndex="0"
+            data-testclass="geneset-expand"
+            data-testid={`${setName}:geneset-expand`}
+            onKeyPress={/* todo_genesets */ () => {}}
+            style={{
+              cursor: "pointer",
+              userSelect: "none",
+            }}
+            onClick={this.onGenesetMenuClick}
+          >
+            <Truncate>
+              <span
+                style={{
+                  maxWidth:
+                    globals.leftSidebarWidth -
+                    120 /* todo_genesets this magic number determines how much of a long geneset name we see, and will be tweaked as we build */,
+                }}
+                data-testid={`${setName}:geneset-label`}
+              >
+                {setName}
+              </span>
+            </Truncate>
+            {isOpen ? (
+              <FaChevronDown
+                data-testclass="geneset-expand-is-expanded"
+                style={{ fontSize: 10, marginRight: 5 }}
+              />
+            ) : (
+              <FaChevronRight
+                data-testclass="geneset-expand-is-not-expanded"
+                style={{ fontSize: 10, marginRight: 5 }}
+              />
+            )}
+          </span>
+          <div>
+            <GenesetMenus genesetsEditable geneset={setName} />
+          </div>
+        </div>
+
+        <div style={{ marginLeft: 15, marginTop: 5, marginRight: 0 }}>
+          {isOpen ? (
+            <Tooltip
+              content="Aggregate all genes in this geneset, and allow coloring by and selecting on the entire set."
+              position={Position.BOTTOM_RIGHT}
+              usePortal
+              hoverOpenDelay={globals.tooltipHoverOpenDelay}
+              modifiers={{
+                preventOverflow: { enabled: false },
+                hide: { enabled: false },
+              }}
+            >
+              <Switch
+                style={{ fontStyle: "italic" }}
+                checked={toggleSummaryHisto}
+                label="Show mean expression across entire set"
+                alignIndicator="left"
+                innerLabel="off"
+                innerLabelChecked="on"
+                onChange={this.toggleSummaryHisto}
+              />
+            </Tooltip>
+          ) : null}
+        </div>
+
+        {isOpen && !toggleSummaryHisto
+          ? _.map(setGenes, (gene) => {
+              return (
+                <Gene
+                  key={gene}
+                  gene={gene}
+                  geneset={setName}
+                  isDiffexp={isDiffexp}
+                />
+              );
+            })
+          : null}
```

Comment:
```suggestion
        {isOpen && !toggleSummaryHisto &&
           _.map(setGenes, (gene) => {
              return (
                <Gene
                  key={gene}
                  gene={gene}
                  geneset={setName}
                  isDiffexp={isDiffexp}
                />
              );
            })}
```
(probably needs linting)